WordPress/Classes/ViewRelated/Activity/ActivityListViewModel.swift

```diff
@@ -156,6 +211,16 @@ class ActivityListViewModel: Observable {
         return formattedDateRanges.joined(separator: " - ")
     }
 
+    func activityTypeDescription() -> String? {
+        if selectedGroups.isEmpty {
+            return NSLocalizedString("Activity Type", comment: "Label for the Activity Type filter button")
+        } else if selectedGroups.count > 1 {
+            return String.localizedStringWithFormat(NSLocalizedString("Activity Type (%d)", comment: "Label for the Activity Type filter button when there are more than 1 activity type selected"), selectedGroups.count)
```

Comment:
To aid in translating we should always be using the explicit positional indexes for localized string formats. 
 
```suggestion
            return String.localizedStringWithFormat(NSLocalizedString("Activity Type ( %1$d)", comment: "Label for the Activity Type filter button when there are more than 1 activity type selected"), selectedGroups.count)
```

Reply:
Thanks! Fixed in 32f969d2f5b405529c5262892595bc6439ce6852